src/main/java/frc/team2412/robot/commands/climb/ExtendFixedHookCommand.java

```diff
@@ -10,6 +10,7 @@
     public ExtendFixedHookCommand(ClimbSubsystem subsystem) {
         this.subsystem = subsystem;
         addRequirements(subsystem);
+        System.out.println("ExtendFixedHookCommand");
```

Comment:
Delete (here & elsewhere). This is already printed out in the CommandScheduler hooks added in Robot::robotInit()